paddle/fluid/platform/enforce.h

```diff
@@ -331,6 +372,12 @@ DEFINE_CUDA_STATUS_TYPE(ncclResult_t, ncclSuccess);
         ::paddle::string::Sprintf(__VA_ARGS__), __FILE__, __LINE__); \
   } while (0)
 
+#define PADDLE_THROW_NANINF(...)                                     \
+  do {                                                               \
+    throw ::paddle::platform::NANINFException(                       \
+        ::paddle::string::Sprintf(__VA_ARGS__), __FILE__, __LINE__); \
+  } while (0)
```

Comment:
Why do you add `PADDLE_THROW_NANINF ` here?

Reply:
Because I only to distinguish NAN/INF exception from the other exceptions during fast debugging for: 
1) handling some exceptions like (summing an empty var) in fast debugging.
2) post-precessing (like saving the parameters and persistable vars).